### DISCUSSION TITLE
On first-invocation, `VM-Add-<etc>` functions are invoked twice

### DISCUSSION THREAD

**Discussion by macintacos** [mg979/vim-visual-multi#233] [Q&A]
This would be a bit difficult to describe, so here's a video of the issue:

https://user-images.githubusercontent.com/8128941/228001763-6fd1e8af-a0b4-4c83-baf5-205f4b3c62b2.mp4

Essentially, I have `<C-j>` mapped to add a cursor down. When first open neovim, it "works", but it seems to do some extra stuff (specifically, it seems to jump to the next line somehow). After that very first invocation, it goes back to normal, and `<C-j>` just adds cursors below like I want it to. And by "first invocation" I really mean "the very first invocation", not "the first invocation every time I use VMM" - it shows up once at the beginning of my neovim session and then goes away completely.

Any ideas on how I could debug this? I don't necessarily think this plugin is "to blame" for this issue, but I can't seem to figure out what is causing it. I do not have duplicate mappings for `<C-j>` as far as I can tell, and I can't really explain why it only happens on the very first invocation and works fine afterwards.

**Comment by mg979** [comment]
Can you open an issue for this? Thanks.
